docs/reference/search/search-your-data/semantic-text-hybrid-search

```diff
@@ -0,0 +1,256 @@
+[[semantic-text-hybrid-search]]
+=== Tutorial: hybrid search with `semantic_text`
+++++
+<titleabbrev>Hybrid search with `semantic_text`</titleabbrev>
+++++
+
+This tutorial demonstrates how to perform hybrid search, combining semantic search with traditional full-text search.
+
+// verification test
```

Comment:
My backport tool also becomes "Unverified" recently and haven't had time to investigate/fix. Let me know if find solution :)

Reply:
@leemthompo I've just become verified! Here are the tutorials I used:

Enabled vigilant mode: [Displaying verification statuses for all of your commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits)
Updated my .zshrc file: [Telling Git about your signing key](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key)

Hope this helps you as well!